strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    fr = Période de paiement
+    da = Intervalbetaling
+    id = Biaya Rutin
+    ko = 주차요금 간격
+    sv = Delvis kostnadsbelagd
+    tr = Süre Ücretli
+    uk = Змінно платне
+    vi = Trả Phí Cách Quãng
+    hu = Idősávos fizetős
+    de = Teilweise gebührenpflichtig
+    fi = Aikamaksu
+    cs = Placeno za čas
+    it = Pagamento a tempo
+    nb = Intervallbetaling
+    zh-Hans = 间隔付款
+    th = จ่ายตามช่วงเวลา
+    ja = 支払間隔
+    ro = Interval de plată
+    ar = دفع عن فترة
+    sk = Platba v intervaloch
+    es = Pago por intervalos
+    pl = Płatny na godziny
+    nl = Intervalbetaling
+
+  [placepage_entrance_number]
+    comment = Entrance number in Place Page
+    en = No.
+    ru = №
+    fr = N°
+    da = Nr.
+    id = Nomor
+    ko = 번호
+    sv = Nr
+    tr = No.
+    uk = №
+    vi = Số
+    hu = Szám
+    de = Nr. der Einfahrt
+    fi = Nro.
+    cs = Č.
+    it = N.
+    nb = Nr.
+    zh-Hans = 编号
+    th = หมายเลข
+    ja = 番号
+    ro = Nr.
+    ar = رقم
+    sk = Číslo
+    es = N.º
+    pl = Nr
+    nl = Nr.
+
+  [placepage_entrance_type]
+    comment = Entrance type in Place Page
+    en = Entrance
+    ru = Подъезд
+    fr = Entrée
+    da = Indgang
+    id = Gerbang Masuk
+    ko = 입구
+    sv = Entré
+    tr = Giriş
+    uk = Під'їзд
+    vi = Lối vào
+    hu = Bejárat
+    de = Typ der Einfahrt
+    fi = Sisäänkäynti
+    cs = Vchod
+    it = Ingresso
+    nb = Vindfang
+    zh-Hans = 入口
+    th = ทางเข้า
+    ja = エントランス
+    ro = Intrare
+    ar = المدخل
+    sk = Vstup
+    es = Entrada
+    pl = Wejście
+    nl = Ingang
+
+  [placepage_flat]
+    comment = Flat number in place page
+    en = app.
+    ru = кв.
+    fr = app.
+    da = app.
+    id = persegi
+    ko = 앱
+    sv = lgh
+    tr = uygulama
+    uk = кв.
+    vi = áp dụng
+    hu = lakás
+    de = Wohnung
+    fi = sov.
+    cs = byt
+    it = app.
+    nb = leil.
+    zh-Hans = 公寓
+    th = ห้อง
+    ja = アプリ
+    ro = ap.
+    ar = تطبيق
+    sk = Byt
+    es = aplicación
+    pl = Mieszkanie
+    nl = app.
+
+  [placepage_open_status]
+    comment = Place status in Place Page
+    en = Open
+    ru = Открыто
+    fr = Ouvrir
+    da = Åbn
+    id = Buka
+    ko = 개점
+    sv = Öppen
+    tr = Açık
+    uk = Відкрито
+    vi = Mở cửa
+    hu = Nyitott
+    de = Geöffnet
+    fi = Auki
+    cs = Otevřeno
+    it = Aperto
+    nb = Åpen
+    zh-Hans = 开放
+    th = เปิด
+    ja = 開く
+    ro = Deschis
+    ar = مفتوح
+    sk = Otvorené
+    es = Abierto
+    pl = Otwarte
+    nl = Open
+
+  [placepage_closed_status]
+    en = Closed
+    ru = Закрыто
+    fr = Fermé
+    da = Lukket
+    id = Tutup
+    ko = 폐점
+    sv = Stängd
+    tr = Kapalı
+    uk = Закрито
+    vi = Đóng cửa
+    hu = Zárt
+    de = Geschlossen
+    fi = Kiinni
+    cs = Zavřeno
+    it = Chiuso
+    nb = Lukket
+    zh-Hans = 已关闭
+    th = ปิด
+    ja = 閉じる
+    ro = Închis
+    ar = مغلق
+    sk = Zatvorené
+    es = Cerrado
+    pl = Zamknięte
+    nl = Gesloten
+
+  [placepage_open_24_7]
```

Comment:
Игорь, меняем или этот перевод тоже нужен?
> On Apr 26, 2016, at 4:33 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61085295>:
> 
> > +    de = Geschlossen
> > +    fi = Kiinni
> > +    cs = Zavřeno
> > +    it = Chiuso
> > +    nb = Lukket
> > +    zh-Hans = 已关闭
> > +    th = ปิด
> > +    ja = 閉じる
> > +    ro = Închis
> > +    ar = مغلق
> > +    sk = Zatvorené
> > +    es = Cerrado
> > +    pl = Zamknięte
> > +    nl = Gesloten
> > +
> > +  [placepage_open_24_7]
> Есть уже готовый перевод [twentyfour_seven]
> Тут еще не понятно зачем расставлены пробелы.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61085295>